src/Raven.Server/Program.cs

```diff
@@ -392,7 +392,7 @@ public static void WriteServerStatsAndWaitForEsc(RavenServer server)
         {
             var workingSetText = PlatformDetails.RunningOnPosix == false ? "working set" : "    RSS    ";
             Console.WriteLine("Showing stats, press any key to close...");
-            Console.WriteLine($"    {workingSetText}     | native mem      | managed mem     | mmap size         | reqs/sec       | docs (all dbs)");
+            Console.WriteLine($"    {workingSetText}     | native mem      | managed mem     | mmap size          | scratch dirty  | reqs/sec       | docs (all dbs)");
```

Comment:
Sanity check. Is this fine? There is an additional space between mmap size and scratch dirty and we are using positions for writing values here.

Reply:
This space is needed